lib/book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_reader :title
+
+  def initialize(book_hash)
+    @author_first_name = book_hash[:author_first_name]
+    @author_last_name = book_hash[:author_last_name]
+    @title = book_hash[:title]
+    @publication_date = book_hash[:publication_date]
+  end
+
+  def author
+    @author_first_name + " " + @author_last_name
+  end
+
+  def publication_year
+    @publication_date[-4..-1]
+  end
```

Comment:
good job in here; the work you did in here could be simplified. We could do the work in these last two methods right in the `initialize` method since we only have to do this one time.